src/services/Servise.js

```diff
@@ -0,0 +1,9 @@
+import { addCustomersAction } from "../redux/store/ItemsReduce"
```

Comment:
don't need to call that file from `UpperCase`
use naming with UpperCase only for `React Components`